frontend/components/EpisodeTemplate.js

```diff
@@ -5,10 +5,10 @@ import {
   CardActionArea,
   CardContent,
   Typography,
-} from '@material-ui/core';
+} from "@material-ui/core";
 import styles from "../styles/Podcast.module.css";
 import Link from "next/link";
-import Image from "next/image"
+import Image from "next/image";
```

Comment:
![image](https://user-images.githubusercontent.com/80564580/188311955-ed3d1dc3-d5bf-4838-8008-e760eb200aaa.png)

Resolve the above error